components/Extensions/src/Markup/Abstract/TextIconExtension.cs

```diff
@@ -33,12 +34,20 @@ protected static FontFamily SymbolThemeFontFamily
     /// <summary>
     /// Gets or sets the thickness of the icon glyph.
     /// </summary>
-    public FontWeight FontWeight { get; set; } = FontWeights.Normal;
+    #if WINUI3
+    public Windows.UI.Text.FontWeight FontWeight { get; set; } = Microsoft.UI.Text.FontWeights.Normal;
+    #elif WINUI2
+    public Windows.UI.Text.FontWeight FontWeight { get; set; } = Windows.UI.Text.FontWeights.Normal;
+    #endif
 
     /// <summary>
     /// Gets or sets the font style for the icon glyph.
     /// </summary>
-    public FontStyle FontStyle { get; set; } = FontStyle.Normal;
+    #if WINUI3
+    public Windows.UI.Text.FontStyle FontStyle { get; set; } = Windows.UI.Text.FontStyle.Normal;
+    #elif WINUI2
+    public Windows.UI.Text.FontStyle FontStyle { get; set; } = Windows.UI.Text.FontStyle.Normal;
+    #endif
```

Comment:
Are these supposed to be the same or similar to the FontWeight change above?

Reply:
WinUI 3 doesn't have a `Microsoft.UI.Text.FontStyle`, checking [the docs](https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.textblock.fontstyle?view=windows-app-sdk-1.5#microsoft-ui-xaml-controls-textblock-fontstyle) it seems like the type is still `Windows.UI.Text.FontStyle`. 

We could remove the compilation conditionals here altogether, or leave them if we think this could change later.